cli/tests/pcluster/cli/test_export_cluster_logs/TestExportClusterLogsCommand/test_helper/pcluster-help.txt

```diff
@@ -18,11 +18,16 @@ options:
   -n CLUSTER_NAME, --cluster-name CLUSTER_NAME
                         Export the logs of the cluster name provided here.
   --bucket BUCKET       S3 bucket to export cluster logs data to. It must be
-                        in the same region of the cluster
+                        in the same region of the cluster. If not specified,
+                        the ParallelCluster default bucket or the
+                        CustomS3Bucket (if specified in the config) will be
+                        used.
   --bucket-prefix BUCKET_PREFIX
                         Keypath under which exported logs data will be stored
                         in s3 bucket. Defaults to <cluster_name>-logs-<current
-                        time in the format of yyyyMMddHHmm>
+                        time in the format of yyyyMMddHHmm>, If not specify
```

Comment:
Typo: "if not specify" => "if not specified"

Reply:
The whole sentence is "If not specify `--bucket` option". Should it be specified?